test/performance/odp_dma_perf.c

```diff
@@ -591,6 +628,7 @@ static odp_pool_t get_src_packet_pool(void)
 	param.pkt.num = prog_conf->num_inflight * prog_conf->num_in_segs * prog_conf->num_sessions;
 	param.pkt.len = prog_conf->src_seg_len;
 	param.pkt.seg_len = prog_conf->src_seg_len;
+	param.pkt.cache_size = 0U;
```

Comment:
Setting cache size to zero will cause global pool access per alloc/free, which may skew results and is probably not the most realistic use case. So it's better to set cache size to a reasonable value (or to add a new command line parameter).

Reply:
Cache sizes improved in v3.